media_driver/linux/common/codec/ddi/media_ddi_encode_hevc.cpp:472:84: error: no member named 'low_delay_seq' in '_VAEncSequenceParameterBufferHEVC::(anonymous struct at /usr/local/include/va/va_enc_hevc.h:202:9)'
    hevcSeqParams->LowDelayMode                       = seqParams->seq_fields.bits.low_delay_seq;
                                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
media_driver/linux/common/codec/ddi/media_ddi_encode_hevc.cpp:473:84: error: no member named 'hierachical_flag' in '_VAEncSequenceParameterBufferHEVC::(anonymous struct at /usr/local/include/va/va_enc_hevc.h:202:9)'
    hevcSeqParams->HierarchicalFlag                   = seqParams->seq_fields.bits.hierachical_flag;
                                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
media_driver/linux/common/codec/ddi/media_ddi_encode_hevc.cpp:559:52: error: no member named 'hierarchical_level_plus1' in '_VAEncPictureParameterBufferHEVC'
    hevcPicParams->HierarchLevelPlus1 = picParams->hierarchical_level_plus1;
                                        ~~~~~~~~~  ^
media_driver/linux/gen12/codec/ddi/media_ddi_decode_hevc_g12.cpp:100:63: error: no member named 'slice_data_num_emu_prevn_bytes' in '_VASliceParameterBufferHEVC'
            codecSlcParams->NumEmuPrevnBytesInSliceHdr = slc->slice_data_num_emu_prevn_bytes;
                                                         ~~~  ^

--- media_driver/linux/common/codec/ddi/media_ddi_encode_hevc.cpp.orig	2019-10-17 14:51:23 UTC
+++ media_driver/linux/common/codec/ddi/media_ddi_encode_hevc.cpp
@@ -469,8 +469,10 @@ VAStatus DdiEncodeHevc::ParseSeqParams(void *ptr)
     hevcSeqParams->SAO_enabled_flag                   = seqParams->seq_fields.bits.sample_adaptive_offset_enabled_flag;
     hevcSeqParams->pcm_enabled_flag                   = seqParams->seq_fields.bits.pcm_enabled_flag;
     hevcSeqParams->pcm_loop_filter_disable_flag       = seqParams->seq_fields.bits.pcm_loop_filter_disabled_flag;
+#if VA_CHECK_VERSION(1, 6, 0)
     hevcSeqParams->LowDelayMode                       = seqParams->seq_fields.bits.low_delay_seq;
     hevcSeqParams->HierarchicalFlag                   = seqParams->seq_fields.bits.hierachical_flag;
+#endif
 
     hevcSeqParams->log2_max_coding_block_size_minus3 = seqParams->log2_diff_max_min_luma_coding_block_size +
                                                        seqParams->log2_min_luma_coding_block_size_minus3;
@@ -556,7 +558,9 @@ VAStatus DdiEncodeHevc::ParsePicParams(
     /* picParams->coding_type; App is always setting this to 0 */
     hevcPicParams->CodingType = picParams->pic_fields.bits.coding_type;
 
+#if VA_CHECK_VERSION(1, 6, 0)
     hevcPicParams->HierarchLevelPlus1 = picParams->hierarchical_level_plus1;
+#endif
 
     /* Reset it to zero now */
     hevcPicParams->NumSlices = 0;
--- media_driver/linux/gen12/codec/ddi/media_ddi_decode_hevc_g12.cpp.orig	2019-11-01 11:00:55 UTC
+++ media_driver/linux/gen12/codec/ddi/media_ddi_decode_hevc_g12.cpp
@@ -97,7 +97,9 @@ VAStatus DdiDecodeHEVCG12::ParseSliceParams(
             }
 
             codecSlcParams->ByteOffsetToSliceData = slc->slice_data_byte_offset;
+#if VA_CHECK_VERSION(1, 6, 0)
             codecSlcParams->NumEmuPrevnBytesInSliceHdr = slc->slice_data_num_emu_prevn_bytes;
+#endif
             codecSlcParams->slice_segment_address = slc->slice_segment_address;
 
             for (i = 0; i < 2; i++)
